packages/zarathustra/skills/asylum_abci_app/strategy.py

```diff
@@ -55,7 +55,6 @@ class AgentPersona(Model):
     """AgentPersona."""
 
     github_username: str
-    role: str
```

Comment:
pretty sure not used at present, happy to revert it should be here ngl but i was in a rush

Reply:
correct, we currently don't use it